src/zac/demo/mor/tests/factories.py

```diff
@@ -0,0 +1,25 @@
+import json
+
+from factory.django import DjangoModelFactory as Dmf
+from zac.accounts.models import User
+from ...models import SiteConfiguration
+
+
+class UserFactory(Dmf):
+
+    class Meta:
+        model = User
+
+    username = 'test'
+    password = 'test1234'
+
+
+class SiteConfigurationFactory(Dmf):
+    class Meta:
+        model = SiteConfiguration
+
+    ztc_catalogus_uuid = 'f7afd156-c8f5-4666-b8b5-28a4a9b5dfc7'
+    global_api_client_id = 'demo-app-KK2CG69NTrSN'
```

Comment:
does not seem wise to be to include that in tests. Is this the actual key? Is the value even relevant in tests? You should not specify fields that aren't required (`blank=False`) in factories.

Reply:
I have to check with Alex with who we decided to include them for the moment